x/subspaces/types/keys.go

```diff
@@ -41,6 +45,8 @@ var (
 	UserPermissionsStorePrefix = []byte{0x05}
 	SectionIDPrefix            = []byte{0x06}
 	SectionsPrefix             = []byte{0x07}
+	UserAllowancePrefix        = []byte{0x08}
+	GroupAllowancePrefix       = []byte{0x09}
```

Comment:
`ExpiringAllowanceQueuePrefix` for removing expired allowances is impossible to implement in 0.45.11 version of cosmos sdk because current `FeeAllowanceI` misses the method called `ExpireAt`. We can implement the feature after updating the cosmos-sdk version over 0.46.

Reference: [0.46.0 FeeAllowanceI](https://github.com/cosmos/cosmos-sdk/blob/v0.46.0/x/feegrant/fees.go#L29) and [0.45.11 FeeAllowanceI](https://github.com/cosmos/cosmos-sdk/blob/v0.45.11/x/feegrant/fees.go#L9)

Reply:
Is there any other way we can have this? Maybe backporting the changes to the `FeeAllowanceI` interface inside our Cosmos SDK fork?

Reply:
Yes, backporting the new version `FeeAllowanceI`  inside our Cosmos SDK fork can solve the issue.